docs/development_suite/set-up-infrastructure/configure-provider.mdx

```diff
@@ -1,258 +1,150 @@
+| Capability        | Providers                                                             |
+| ----------------- |:---------------------------------------------------------------------:|
+| Git Provider      | GitLab, GitHub, Bitbucket, Azure DevOps                               |
+| Secret Manager    | GitLab, Vault                                                         |
+| CI/CD Tool        | GitLab (Git Runner), GitHub (GitHub Actions), Jenkins, Azure DevOps   |
 
-## Providers APIs
-The Console currently exposes two APIs:
-- POST - `/api/backend/tenants/{tenantId}/providers`: used to create a new provider;
-- PATCH - `/api/backend/tenants/{tenantId}/providers/{providerId}`: used to edit an existing provider.
+## Providers Overview
 
-:::note
-The `tenantId` refers to the identifier of the company you need to associate the provider with.
-The `providerId` refers to the identifier of the provider created earlier.
-:::
+Mia-Platform Console allows authorized users to manage providers. More specifically, a user can:
 
-:::info
-These APIs are protected and can be used only if you belong to the group `access_token_manager`.
-:::
+* Add a new connection to a provider;
+* Edit the connection to a provider.
+
+To access these features, head to the Company Overview area of the Console by clicking the related button on the home page, next to your company name.
+
+![Go to Company Overview](img/go-to-company-overview.png)
+
+By clicking on "Providers" you will be redirected to the related section, in which (assuming you already have some providers connected) you can see a table containing some information about the providers connected to the Console:
+
+* **Name**: provider name, assigned by the user while connecting the provider to the Console;
+* **Capabilities**: provider capabilities (Git Provider, Secret Manager, CI/CD Tool);
+* **Service**: external reference service on which the provider relies to provide its capabilities (e.g. Azure, GitHub, Jenkins).
+
+Through the search bar at the top of the table you will be able to filter data based on provider name or service, while the bar next to it allows you to filter by one or more capabilities.
+
+![Providers overview](img/providers-overview.png)
+
+## Connect a Provider
+
+You can connect a provider by clicking on the "Add provider connection" button, located in the page's top right corner. A modal will show up, and, through a few steps, you will be asked to enter some information related to the provider.
+
+### Step 1: Provider Services
+
+![Add provider 1a](img/add-provider-1a.png)
+![Add provider 1b](img/add-provider-1b.png)
+
+In this step, you are required to choose the specific provider you want to use among the ones supported by the Console and, for the chosen provider, the capabilities you want to use it for ("Git Provider", "Secret Manager", "CI/CD Tool").
```

Comment:
Qua magari speficicherei che in caso il proovider abbia solo una capability ovviamente non potrà scegliere nlla nello step 1b